docs/organizations/using-a-coding-standard.md

```diff
@@ -22,23 +22,23 @@ To create a coding standard for your organization:
 
 1.  Name your coding standard and click **Create standard**.
 
-    Optionally, select a repository that Codacy will use as a baseline to bootstrap the tool and pattern configurations for the new coding standard. This is useful if you already have a repository with code patterns configured that you wish to use as a template for creating the coding standard.
+    Optionally, select a repository for Codacy to use as a baseline when bootstrapping the tool and pattern configurations for the new coding standard. This is useful if you already have a configured repository that you wish to use as a template.
 
     ![Creating a new coding standard](images/coding-standard-create.png)
 
-1.  Select all the programming languages that the new coding standard will cover and click **Next: Tools and patterns**.
+1.  Select the programming languages that the new coding standard should cover and click **Next: Tools and patterns**.
 
     The coding standard will only include configurations for the tools that support at least one of the selected languages.
 
     ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
 
 1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.
 
-    -   Toggle the tools that Codacy will run when analyzing your code
-    -   For each enabled tool, configure the code patterns that the tool will use
+    -   Toggle the tools to run when analyzing your code
+    -   For each enabled tool, configure the code patterns to use
```

Comment:
Again, it's a bit more concise like this—unless we want to explicitly mention Codacy